Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py

```diff
@@ -73,6 +82,8 @@
 upgradeProperties['2023D2PU']['ScenToRun'] = ['GenSimFull','DigiFullPU','RecoFullGlobalPU', 'HARVESTFullGlobalPU']
 upgradeProperties['2023D3PU'] = deepcopy(upgradeProperties['2023D3'])
 upgradeProperties['2023D3PU']['ScenToRun'] = ['GenSimFull','DigiFullPU','RecoFullGlobalPU', 'HARVESTFullGlobalPU']
+upgradeProperties['2023D4PU'] = deepcopy(upgradeProperties['2023D4'])
+upgradeProperties['2023D4PU']['ScenToRun'] = ['GenSimFull','DigiFullPU','RecoFullGlobalPU', 'HARVESTFullGlobalPU']
```

Comment:
Just noticed - here you have 2023D4PU through global reco, while 2023D4 is only through local reco